src/commands/io-services/create-demo.ts

```diff
@@ -0,0 +1,261 @@
+        },
+        () => {
+          cli.action.stop(
+            chalk.green(`Message successfully sent to ${sandboxFiscalCode}`)
+          );
+        }
+      )
+      .run();
+  }
+
+  private getAdminApiClient = () =>
+    AdminApiClient(
+      getRequiredStringEnv("BASE_URL_ADMIN"),
+      getRequiredStringEnv("OCP_APIM")
+    );
+
+  private getServicesApiClient = () =>
+    ServicesApiClient(
+      getRequiredStringEnv("BASE_URL_ADMIN"),
+      getRequiredStringEnv("OCP_APIM")
+    );
+
+  private createUser = (user: UserPayload): TaskEither<Error, UserCreated> =>
+    new TaskEither(
+      new Task(() => this.getAdminApiClient().createUser({ userPayload: user }))
+    )
+      .mapLeft(errorsToError)
+      .chain(
+        fromPredicate(
+          response => response.status === 200,
+          () => Error("Could not create user")
+        )
+      )
+      .chain(response =>
+        fromEither(UserCreated.decode(response.value)).mapLeft(errorsToError)
+      );
+
+  private updateUserGroups = (
+    email: EmailAddress,
+    userGroups: ReadonlyArray<string>
+  ): TaskEither<Error, GroupCollection> =>
+    new TaskEither(
+      new Task(() =>
+        this.getAdminApiClient().updateGroups({
+          email,
+          userGroupsPayload: {
+            groups: userGroups
+          }
+        })
+      )
+    )
+      .mapLeft(errorsToError)
+      .chain(
+        fromPredicate(
+          response => response.status === 200,
+          () => Error("Could not update groups")
+        )
+      )
+      .chain(response =>
+        fromEither(GroupCollection.decode(response.value)).mapLeft(
+          errorsToError
+        )
+      );
+
+  private createSubscription = (
+    email: string,
+    subscriptionId: string,
+    productName: NonEmptyString
+  ): TaskEither<Error, Subscription> =>
+    new TaskEither(
+      new Task(() =>
+        this.getAdminApiClient().createSubscription({
+          email,
+          subscription_id: subscriptionId,
+          productNamePayload: { product_name: productName }
+        })
+      )
+    )
+      .mapLeft(errorsToError)
+      .chain(
+        fromPredicate(
+          response => response.status === 200,
+          () => Error(`Could not create the subscription ${subscriptionId}`)
+        )
+      )
+      .chain(response =>
+        fromEither(Subscription.decode(response.value)).mapLeft(errorsToError)
+      );
+
+  private createService = (
+    service: AdminService
+  ): TaskEither<Error, AdminService> =>
+    new TaskEither(
+      new Task(() => this.getAdminApiClient().createService({ service }))
+    )
+      .mapLeft(errorsToError)
+      .chain(
+        fromPredicate(
+          response => response.status === 200,
+          () => Error(`Could not create the service ${service.service_id}`)
+        )
+      )
+      .chain(response =>
+        fromEither(AdminService.decode(response.value)).mapLeft(errorsToError)
+      );
+
+  private submitMessageforUser = (
```

Comment:
we just chose to not send any email to the user as the fake fiscal code will be a fixed value. 
can you delete the email sending part ? (sorry it was decided yesterday night : )